ci/upload-bazel-metrics.yml

```
@@ -46,12 +46,15 @@ steps:
         git_current_commit: "'"$(git rev-parse HEAD)"'",
         git_current_tree: "'"$(git rev-parse HEAD:./)"'",
       }'
-      jq <job-md.json .
-      gcs "$GCRED" cp "job-md.json" "gs://daml-data/bazel-metrics/$(pipeline_id)/job-md.json"
-      [[ -f "build-profile.json" ]] && gcs "$GCRED" cp "build-profile.json" "gs://daml-data/bazel-metrics/$(pipeline_id)/build-profile.json" || echo "build-profile.json not found"
-      [[ -f "build-events.json" ]] && gcs "$GCRED" cp "build-events.json" "gs://daml-data/bazel-metrics/$(pipeline_id)/build-events.json" || echo "build-events.json not found"
-      [[ -f "test-profile.json" ]] && gcs "$GCRED" cp "test-profile.json" "gs://daml-data/bazel-metrics/$(pipeline_id)/test-profile.json" || echo "test-profile.json not found"
-      [[ -f "test-events.json" ]] && gcs "$GCRED" cp "test-events.json" "gs://daml-data/bazel-metrics/$(pipeline_id)/test-events.json" || echo "test-events.json not found"
+      target_dir="$(Build.StaginDirectory)/$(pipeline_id)"
+      cp "job-md.json" "$target_dir/job-md.json"
+      [[ -f "build-profile.json" ]] && cp "build-profile.json" "$target_dir/build-profile.json" || echo "build-profile.json not found"
+      [[ -f "build-events.json" ]] && cp "build-events.json" "$target_dir/build-events.json" || echo "build-events.json not found"
+      [[ -f "test-profile.json" ]] && cp "test-profile.json" "$target_dir/test-profile.json" || echo "test-profile.json not found"
+      [[ -f "test-events.json" ]] && cp "test-events.json" "$target_dir/test-events.json" || echo "test-events.json not found"
```

You could do this in a loop if you like. Not sure if it's more or less pleasant to read.

```suggestion
      for log_file in 'build-profile.json' 'build-events.json' 'test-profile.json' 'test-events.json'; do
        [[ -f "$log_file" ]] && cp "$log_file" "$target_dir/$log_file" || echo "$log_file not found"
      done
```

---

I like that. Thanks!